src/Tests/Toolchain/ToolchainProvider.test.ts

```diff
@@ -34,28 +34,22 @@ import {
   MockCompilerWithMultipleInstalledToolchains,
   MockCompilerWithNoInstalledToolchain,
 } from "../MockCompiler";
-import { OneCompiler } from "../../Backend/One/OneToolchain";
 
 suite("Toolchain", function () {
-  const oneToolhcainEnv = new ToolchainEnv(new OneCompiler());
   const oneBackendName = "ONE";
+  const edgeTPUBackendName = "EdgeTPU";
+  const backendName = "dummy_backend";
   const compiler = new MockCompiler();
   const toolchainEnv = new ToolchainEnv(compiler);
-  const backendName = "dummy_backend";
 
   setup(function () {
-    Object.keys(gToolchainEnvMap).forEach(
-      (key) => delete gToolchainEnvMap[key]
-    );
-
-    gToolchainEnvMap[oneBackendName] = oneToolhcainEnv;
     gToolchainEnvMap[backendName] = toolchainEnv;
   });
 
   teardown(function () {
-    Object.keys(gToolchainEnvMap).forEach(
-      (key) => delete gToolchainEnvMap[key]
-    );
+    if (gToolchainEnvMap[backendName] !== undefined) {
+      delete gToolchainEnvMap[backendName];
+    }
   });
```

Comment:
I think teardown function should be located at the bottom of "Toolchain" test suite :-D
(related to https://github.com/Samsung/ONE-vscode/pull/1686#discussion_r1336789547)

Reply:
Oh, Yes.
Thanks for reminding!

I'll fix the codes. 😃